zerver/webhooks/airbrake/tests.py

```diff
@@ -5,6 +5,9 @@ class AirbrakeHookTests(WebhookTestCase):
     STREAM_NAME = "airbrake"
     URL_TEMPLATE = "/api/v1/external/airbrake?stream={stream}&api_key={api_key}"
     WEBHOOK_DIR_NAME = "airbrake"
+    # We ensure that even we don't register any events in the view function
+    # this WebhookTestCase will not raise an exception when send_webhook_message is called
+    VIEW_FUNCTION_NAME = "api_airbrake_webhook"
```

Comment:
Hmm, I don't see an exception if I delete this line and run tests.

Reply:
This line is meant for test coverage in `zerver/lib/test_classes.py` at line 1412
```
if all_event_types is None:
        return
```